build.gradle.kts

```diff
@@ -27,7 +27,8 @@ subprojects {
         val testImplementation by configurations
         val testRuntimeOnly by configurations
 
-        implementation(platform("org.springframework.boot:spring-boot-dependencies:2.7.4"))
+        implementation(enforcedPlatform("org.springframework.boot:spring-boot-dependencies:2.7.5"))
```

Comment:
try without spring-boot-dependencies. It's already import in chutney parent

Reply:
✓